code/datums/status_effects/buffs.dm

```diff
@@ -38,4 +38,9 @@
 
 /datum/status_effect/void_price/tick()
 	playsound(owner, 'sound/weapons/bite.ogg', 50, 1)
-	owner.adjustBruteLoss(3)
\ No newline at end of file
+	owner.adjustBruteLoss(3)
+
+/datum/status_effect/exercised
+	id = "Exercised"
+	duration = 1200
+	alert_type = null
```

Comment:
Hm. Might want to check on how status effects are handled here vs there. They have this removed from fast processing and added to regular processing.